### PR TITLE
Push user signed up event to posthog

### PR DESCRIPTION
**Pull request by PierreOucif**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -10,4 +10,6 @@ INTERNAL_API_KEY=internal-api-key
 IMAGE_BUCKET_NAME=develop-onlydust-app-images
 BACKOFFICE_WEBHOOK_URL=https://enb2xwybfuwfc.x.pipedream.net/
 BACKOFFICE_API_KEY-api-key-95-111-47
-AUTH0_JWKS_URL=https://develop-onlydust.eu.auth0.com/
\ No newline at end of file
+AUTH0_JWKS_URL=https://develop-onlydust.eu.auth0.com/
+POSTHOG_API_KEY=phc_bJnXvrlhdUCQUTlA9TZCuKXZt7xh5mVR71faqE4i6cb
```
Comment (AnthonyBuisset) [inline .env.example:14]: ggshield ne va pas raler ?

File: pom.xml [hunk 2]
```diff
@@ -13,6 +13,7 @@
         <module>backoffice-api-contract</module>
         <module>accounting-domain</module>
         <module>kernel</module>
+        <module>infrastructure/posthog-api-client-adapter</module>
```
Comment (AnthonyBuisset) [inline pom.xml:16]: `posthog-api-client-adapter` should be a child of `infrastructure` right ?

**Review by AnthonyBuisset — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (PierreOucif) [reply .env.example:14]: A priori non

File: pom.xml (hunk 2 above)
Comment (PierreOucif) [reply pom.xml:16]: Bien vu merci !

**Pull request merged**